app/presenters/order_detail_presenter.rb

```diff
@@ -3,6 +3,7 @@ class OrderDetailPresenter < SimpleDelegator
   include ActionView::Helpers::NumberHelper
   include DateHelper
   include Rails.application.routes.url_helpers
+  include PriceDisplayment
```

Comment:
I think that's a great idea, but I'll make a separate task for it if that's cool.